crates/miden-lib/src/account/mod.rs

```diff
@@ -1,6 +1,6 @@
 use super::auth::AuthScheme;
 
 pub mod auth;
-pub(super) mod components;
+pub mod components;
```

Comment:
Is this still required? I thought the idea was that we expose these via the component types like `BasicWallet`, etc.

Reply:
The reason for exposing this is that for now we were building the templates from the client (for https://github.com/0xPolygonMiden/miden-client/issues/695) and for this we need access to the libraries. If we decide to distribute these templates from `miden-base`, this would not be needed.

Reply:
Ah, makes sense!